plugins/com.google.cloud.tools.eclipse.appengine.whitelist/pom.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.tools.eclipse</groupId>
+    <artifactId>trunk</artifactId>
+    <version>94.0.0-SNAPSHOT</version>
+    <relativePath>../../</relativePath>
+  </parent>
+  <groupId>com.google.cloud.tools.eclipse</groupId>
+  <artifactId>com.google.cloud.tools.eclipse.appengine.whitelist</artifactId>
+  <version>94.0.0-SNAPSHOT</version>
+  <packaging>eclipse-plugin</packaging>
+  
+  <dependencies>
```

Comment:
Is `junit` necessary?  Are you going to have tests defined in-bundle?  That's not the usual case for bundles.